backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt

```diff
@@ -9,6 +10,8 @@ import org.jetbrains.exposed.dao.id.IntIdTable
 object Administrators : IntIdTable() {
     val email = varchar("email", 100).uniqueIndex()
     val projectId = reference("projectId", Projects)
+    val regionId = optReference("regionId", Regions)
+    val role = varchar("role", 50)
```

Comment:
16 should be enough I think, 50 is fine too

Reply:
Yeah. let's do 32 :D With 16, I am too afraid we will come up with long role names :D